client/src/templates/Challenges/components/challenge-title.css

```diff
@@ -41,7 +41,7 @@
 }
 
 .challenge-title-breadcrumbs ol a:focus:not(:focus-visible) {
-  outline: none;
+  outline-color: transparent;
```

Comment:
```suggestion
```

Revert back to outline: none. This is for non-keyboard focus in which we don't want a focus outline.